data_structure/tests/test_array.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+from data_structure.array import *
+
+
+class TestPlusOne(unittest.TestCase):
+    def test_plusOne_one(self):
+        output = PlusOne().plusOne([1, 2, 9])
+        ans = [1, 3, 0]
+        self.assertEqual(ans, output)
```

Comment:
you want to have something like:
```
self.assertEqual(PlusOne.plusOne([1, 2, 9], [1, 3, 0])
```

so it's easier to read between the 2. Same goes to all the other test cases

Reply:
thank you! although line gets really long when it comes to:
        self.assertEqual([True, True, True, False, True], KidsWithCandies().kidsWithCandies([2, 3, 5, 1, 3], 3))
I may keep the current format for more readability (for me at least)

Reply:
if you break that line down into multiple lines like this, then it becomes readable:

```
self.assertEqual(
    KidsWithCandies().kidsWithCandies([2, 3, 5, 1, 3], 3),
    [True, True, True, False, True]
)
```